packages/emotion/src/withStyle.tsx

```diff
@@ -29,9 +29,9 @@ import type {
   RefAttributes
 } from 'react'
 
-import isEqual from 'lodash/isEqual'
 import hoistNonReactStatics from 'hoist-non-react-statics'
 
+import { deepEqual as isEqual } from '@instructure/ui-utils'
```

Comment:
Removed the `isEqual` import and usage since we have our "own" (which is just a wrapper around `fast-deep-equal`) which is supposed to be much more [performant](https://www.npmjs.com/package/fast-deep-equal#performance-benchmark).